vortex-tui/src/convert.rs

```diff
@@ -19,12 +19,19 @@ use vortex::file::VortexWriteOptions;
 use vortex::stream::{ArrayStream, ArrayStreamArrayExt};
 use vortex::{Array, ArrayRef};
 
+#[derive(Default)]
+pub struct Flags {
+    pub quiet: bool,
```

Comment:
Isn't it usually the other way around?!

Reply:
I think usually you want the print. See curl or wget for example